.gitlab-ci.yml

```diff
@@ -235,7 +235,6 @@ sourcemaps_preview:
     policy: pull
   environment: "preview"
   script:
-    - yarn run build:webpack:preview -- --devtool source-map
     - DATADOG_API_KEY="$(get_secret 'dd_synthetic_api_key_prod')" ./node_modules/.bin/datadog-ci sourcemaps upload ./public/static --service docs --minified-path-prefix "https://docs-staging.datadoghq.com/${CI_COMMIT_REF_NAME}/static/" --release-version "${CI_COMMIT_SHORT_SHA}"
```

Comment:
Can we build the source map with esbuild standalone if there isn't an option with the one bundled with hugo? https://esbuild.github.io/api/#sourcemap